<launch>
	<group ns="turtlesim1">
            <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
            <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>
            <node name="turtle1_tf_broadcaster" pkg="rviz_pose_publisher" type="tf_broadcaster.py" respawn="false" output="screen" >
                <param name="turtle" type="string" value="turtle1" />
            </node>
	</group>
	<group ns="turtlesim2">
           <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
            <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>	</group>
            <node name="turtle2_tf_broadcaster" pkg="rviz_pose_publisher" type="tf_broadcaster.py" respawn="false" output="screen" >
                <param name="turtle" type="string" value="turtle1" /> 
            </node>
</launch>
